src/services/dispatchers/dispatchers/types.ts

```diff
@@ -71,13 +91,27 @@ export interface DispatchersService {
 }
 
 export interface Dispatcher {
-  id?: ObjectId;
+  id: ObjectId;
+  /** The unique name of the dispatcher */
+  name?: string;
+  /** A description of the dispatcher */
+  description?: string;
   /** The type of event the Dispatcher will respond to e.g 'user_deleted' */
   eventType: string;
   /** The actions the Dispatcher shall execute */
   actions: Array<Action>;
+  /** A list of string identifiers that can be attached to a Dispatcher */
+  tags?: Array<string>;
   /** The creation timestamp of the Dispatcher */
-  creationTimestamp?: Date;
+  creationTimestamp: Date;
   /** The update timestamp of the Dispatcher */
-  updateTimestamp?: Date;
+  updateTimestamp: Date;
 }
+
+export type CreateDispatcherBody = Omit<
```

Comment:
I think to match our interface naming convention, these should be `DispatcherCreation` and `DispatcherUpdate` respectively?